components/alert.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alert</title>
+</head>
+<script src="https://kit.fontawesome.com/6b52d0ce66.js" crossorigin="anonymous"></script>
+<body>
+    <div class="all_alert">
+
+      
+        <div>
+            <i class="fas fa-exclamation-circle"></i>
+            <div class="alert">
+                This is an error alert</div>
+        </div>
+        <div>
+            <i class="fas fa-exclamation-triangle"></i>
+            <div class="alert">This is a warning alert</div>
+        </div>
+        <div>
+            <i class="far fa-check-circle"></i>
+            <div class="alert">This is a success alert</div>
+        </div>
+        <div>
+            <i class="fas fa-info"></i>
+            <div class="alert">This is info alert</div>
+        </div>
+        
+        
+       
+       
+
```

Comment:
A suggestion here would be to try avoiding so much empty space.

Reply:
okk